Balloon/sys/public.h

```diff
@@ -12,7 +12,8 @@ DEFINE_GUID(GUID_DEVINTERFACE_BALLOON,
 #define VIRTIO_BALLOON_S_MINFLT   3   /* Number of minor faults */
 #define VIRTIO_BALLOON_S_MEMFREE  4   /* Total amount of free memory */
 #define VIRTIO_BALLOON_S_MEMTOT   5   /* Total amount of memory */
-#define VIRTIO_BALLOON_S_NR       6
+#define VIRTIO_BALLOON_S_AVAIL    6   /* Available memory */
```

Comment:
I think I had convinced myself that there's no need to do anything for counters that are available only with or only without USE_BALLOON_SERVICE.

The driver initializes the entire stats with -1's

  `RtlFillMemory (devCtx->MemStats, sizeof (BALLOON_STAT) * VIRTIO_BALLOON_S_NR, -1);`

and -1 means 'unset'. See reset_stats() in hw/virtio/virtio-balloon.c and the way s->stats is filled in virtio_balloon_receive_stats.